<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>XIVN1987</author>
 <class>USBHID</class>
 <widget class="QWidget" name="USBHID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>USBHID - DAC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/Twiiter.png</normaloff>image/Twiiter.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="vLayout">
   <item>
    <layout class="QHBoxLayout" name="hLayout2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="grpChnl">
       <property name="title">
        <string>通道</string>
       </property>
       <layout class="QHBoxLayout" name="hLayoutC">
        <item>
         <widget class="QRadioButton" name="rdoCH1">
          <property name="text">
           <string>CH1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdoCH2">
          <property name="text">
           <string>CH2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdoCH3">
          <property name="text">
           <string>CH3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdoCH4">
          <property name="text">
           <string>CH4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpWave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>波形</string>
       </property>
       <layout class="QHBoxLayout" name="hLayoutW">
        <item>
         <widget class="QComboBox" name="cmbWave">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSend">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtMain">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>产品名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>226</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>打开连接</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
